### ISSUE TITLE
A11y Dash: Group a11y defect columns

### ISSUE THREAD

**Issue by JoeTice** [department-of-veterans-affairs/va.gov-team#26663]
## Issue Description
Group team open/closed a11y defects together, for example the [Average Number of Individual Tests Passing, Pending, and Failing](https://va-gov.domo.com/page/1232474697/kpis/details/1922031625) visualization in the E2E dashboard has days grouped together with a gray background in order to separate each day from the others adjacent to it. Something similar makes sense for this card as well.

![image.png](https://images.zenhubusercontent.com/5d2633e0a6db635dbcbaf8d5/c2c819b4-ed59-4e15-afc1-2cb430f0e7ee)
---
## Tasks
- [ ] Group columns as discussed

## Acceptance Criteria
- [ ] Review w/ team when complete

**Comment by rjohnson2011** [comment]
Revised Defects and Open/Closed/Total Issues by Teams charts in Domo, now importing data entirely from SQL.

**Comment by rjohnson2011** [comment]
Refactored the SQL queries so team open/closed accessibility issues are grouped together. Finalizing labels in Domo to wrap up early tomorrow.

**Comment by rjohnson2011** [comment]
Defects are now grouped in Domo: 
![image](https://user-images.githubusercontent.com/72466113/124300355-7d2bd280-db2c-11eb-9f5d-3d7fe3d7f9c9.png)
